vowpalwabbit/action_score.cc

```diff
@@ -43,10 +43,10 @@ std::string to_string(const ACTION_SCORE::action_scores& action_scores_or_probs,
   std::string delim;
   for (const auto& item : action_scores_or_probs)
   {
-    ss << delim << fmt::format("{},{}", item.action, VW::fmt_float(item.score, decimal_precision));
+    ss << delim << fmt::format("{}:{}", item.action, VW::fmt_float(item.score, decimal_precision));
```

Comment:
if all tests pass then we have missed a test

Reply:
I agree, I don't think tests will pass though

Reply:
Actually, yep I think we're missing some tests here

Reply:
So it is covered by tests, but it seems like the harness is failing to see the diff here